decrepit/xts/xts_test.cc

```diff
@@ -206,6 +208,245 @@ static const XTSTestCase kXTSTestCases[] = {
         "dfce93b6f82ae22423274e58a0821cc5502e2d0ab4585e94de6975be5e0b4efce51cd3"
         "e70c25a1fbbbd609d273ad5b0d59631c531f6a0a57b9",
     },
+    // https://github.com/BrianGladman/modes/blob/master/testvals/xts.6#L313
+    // VEC 30, len = 16 bytes = 1 block
+    {
+        "fffefdfcfbfaf9f8f7f6f5f4f3f2f1f0efeeedecebeae9e8e7e6e5e4e3e2e1e0"
+        "bfbebdbcbbbab9b8b7b6b5b4b3b2b1b0afaeadacabaaa9a8a7a6a5a4a3a2a1a0",
+        "9a785634120000000000000000000000",
+        "000102030405060708090a0b0c0d0e0f",
+        "c30ca8f2ed57307edc87e544867ac888",
+    },
+    // https://github.com/BrianGladman/modes/blob/master/testvals/xts.6#L321
+    // VEC 31, len = 17 bytes = 1 block + 1 byte
+    {
+        "fffefdfcfbfaf9f8f7f6f5f4f3f2f1f0efeeedecebeae9e8e7e6e5e4e3e2e1e0"
+        "bfbebdbcbbbab9b8b7b6b5b4b3b2b1b0afaeadacabaaa9a8a7a6a5a4a3a2a1a0",
+        "9a785634120000000000000000000000",
+        "000102030405060708090a0b0c0d0e0f10",
+        "7f117752cc598a8b0d81d88af9f9bec8c3",
+    },
+    // https://github.com/BrianGladman/modes/blob/master/testvals/xts.6#L361
+    // VEC 36, len = 22 bytes = 1 block + 6 bytes
+    {
+        "fffefdfcfbfaf9f8f7f6f5f4f3f2f1f0efeeedecebeae9e8e7e6e5e4e3e2e1e0"
+        "bfbebdbcbbbab9b8b7b6b5b4b3b2b1b0afaeadacabaaa9a8a7a6a5a4a3a2a1a0",
+        "9a785634120000000000000000000000",
+        "000102030405060708090a0b0c0d0e0f101112131415",
+        "75e8188bcce59ada939f57de2cb9a489c30ca8f2ed57",
+    },
+    // https://github.com/BrianGladman/modes/blob/master/testvals/xts.6#L433
+    // VEC 45, len = 31 bytes = 1 block + 15 bytes
+    {
+        "fffefdfcfbfaf9f8f7f6f5f4f3f2f1f0efeeedecebeae9e8e7e6e5e4e3e2e1e0"
+        "bfbebdbcbbbab9b8b7b6b5b4b3b2b1b0afaeadacabaaa9a8a7a6a5a4a3a2a1a0",
+        "9a785634120000000000000000000000",
+        "000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e",
+        "581ea1fee5516ad432ddebe75fd27c6fc30ca8f2ed57307edc87e544867ac8",
+    },
+    // https://github.com/BrianGladman/modes/blob/master/testvals/xts.12#L2611
+    // VEC 301, len = 32 bytes = 2 blocks (decryption vector)
+    // This vector did not pass, the output below is the same for
+    // the C implementation, |CRYPTO_xts128_encrypt|, and the AArch64 assembly.
```

Comment:
what does this mean?

Reply:
The ones that were added by the original author were covering different plaintext lengths but for key size 128 which we don't claim to support in the API (though I believe the C and Asm code do support it).